etcdctl/ctlv3/command/printer_json.go

```diff
@@ -59,6 +66,60 @@ func printJSON(v interface{}) {
 	fmt.Println(string(b))
 }
 
+func printEndpointStatusWithHexJSON(r []epStatus) {
+	var buffer bytes.Buffer
+	var b []byte
+
+	buffer.WriteString("[")
```

Comment:
Please avoid "hand-crafting" of JSON format as it might have subtle escaping consequences. 

Please see discussion in: 
https://github.com/etcd-io/etcd/pull/12595#discussion_r561638901